src/toolchain/toolchain.ts

```diff
@@ -54,6 +76,12 @@ export class SwiftToolchain {
 
     logDiagnostics(channel: SwiftOutputChannel) {
         channel.logDiagnostic(`Toolchain Path: ${this.toolchainPath}`);
+        if (this.defaultTarget) {
+            channel.logDiagnostic(`Default Target Triple: ${this.defaultTarget}`);
```

Comment:
It is not completely obvious what "Default Target Triple" is

Reply:
Well, it’s just for logging now — this is what `swift --version` outputs for the second line, indicating the default build target the driver will use. We can possibly eliminate "triple" here if you think that’s better.

Targets are tied to SDKs, so it’s actually more related to #256. I put it here because it uses `-print-target-info` output.

Reply:
Don't think triple means much to many people

Reply:
It’s useless for most people — but it shouldn’t be visible to most people too. The diagnostic output is disabled by default.